src/App.vue

```diff
@@ -1,28 +1,25 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <Calendar />
+    <Footer />
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
+import Footer from "./components/Footer";
+import Calendar from "./components/Calendar";
 
 export default {
   name: 'App',
   components: {
-    HelloWorld
-  }
-}
+    Footer,
+    Calendar
+  },
+};
 </script>
 
 <style>
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
```

Comment:
let us retain these two lines - those help with some font rendering issues in Chrome and Firefox respectively:

```css
  -webkit-font-smoothing: antialiased; 
  -moz-osx-font-smoothing: grayscale;
```